src/main/scala/sbtassembly/Shader.scala

```diff
@@ -81,7 +81,16 @@ private[sbtassembly] object Shader {
     }}
 
     val proc = JJProcessor(jjrules, verbose = level == Level.Debug, true)
-    val files = AssemblyUtils.getMappings(dir, Set())
+
+    /*
+    jarjar MisplacedClassProcessor class transforms byte[] to a class using org.objectweb.asm.ClassReader.getClassName
+    which always translates class names containing '.' into '/', regardless of OS platform.
+    We need to transform any windows file paths in order for jarjar to match them properly and not omit them.
+     */
+//    val files = AssemblyUtils.getMappings(dir, Set())
+
+    val files = AssemblyUtils.getMappings(dir, Set()).map(f =>
+      if (f._2.contains('\\')) (f._1, f._2.replace('\\', '/')) else f)
```

Comment:
Problem is that if the string is on a Windows path, it'll currently throw a `MatchError`:

```
scala> val PathList(xs) = "com\\clicktale\\ai"
scala.MatchError: com\clicktale\ai (of class java.lang.String)
  ... 33 elided

scala>
```